File: Task-3/New-Array.js
// Function to concatenate arrays A and B
function concatenateArrays(N, A, B) {
  const C = [...B, ...A];
  return C;
}

// Function to process input string
function processInput(input) {
  const lines = input.trim().split("\n");
  const N = parseInt(lines[0]);
  const A = lines[1].split(" ").map(Number);
  const B = lines[2].split(" ").map(Number);

  return concatenateArrays(N, A, B);
}

// Example usage
const input = "2\n1 2\n3 4";
const result = processInput(input);
console.log(result.join(" "));

File: Task-3/New Array.c
#include <stdio.h>

void concatenateArrays(int N, int A[], int B[], int C[])
{
    for (int i = 0; i < N; i++)
    {
        C[i] = B[i];
    }

    for (int i = 0; i < N; i++)
    {
        C[i + N] = A[i];
    }
}

int main()
{
    int N;
    scanf("%d", &N);

    int A[N], B[N], C[2 * N];

    for (int i = 0; i < N; i++)
    {
        scanf("%d", &A[i]);
    }

    for (int i = 0; i < N; i++)
    {
        scanf("%d", &B[i]);
    }

    concatenateArrays(N, A, B, C);

    for (int i = 0; i < 2 * N; i++)
    {
        printf("%d", C[i]);
        if (i < 2 * N - 1)
        {
            printf(" ");
        }
    }

    return 0;
}
